pkg/compute/rust.go

```diff
@@ -54,12 +54,13 @@ func (m *CargoManifest) Read(filename string) error {
 // CargoMetadata models information about the workspace members and resolved
 // dependencies of the current package via `cargo metadata` command output.
 type CargoMetadata struct {
-	Package []CargoPackage `json:"packages"`
+	Package         []CargoPackage `json:"packages"`
+	TargetDirectory string         `json:"target_directory"`
 }
 
 // Read the contents of the Cargo.lock file from filename.
 func (m *CargoMetadata) Read() error {
-	cmd := exec.Command("cargo", "metadata", "--format-version", "1")
+	cmd := exec.Command("cargo", "metadata", "--quiet", "--format-version", "1")
```

Comment:
What is the reason for adding the `--quiet` flag here?

Reply:
It suppresses some warnings, esp. for workspaces, like:

```
$ cargo metadata --format-version 1 > /dev/null
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /Users/goro-fuji/ghq/github.com/gfx/ecp-hello/app/Cargo.toml
workspace: /Users/goro-fuji/ghq/github.com/gfx/ecp-ecp-hello/Cargo.toml
```

It doesn't affect the fastly cli but I feel it's better to suppress.